modules/browser_object_navigation.py

```diff
@@ -0,0 +1,109 @@
+from pypom import Page
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webelement import WebElement
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class Navigation(Page):
```

Comment:
Is this class really a page, or more of a Region?

Reply:
Also, you didn't create an `__init__` here, why is that?

Reply:
I'm considering major parts of the chrome as Pages because otherwise I'll need to model the entire chrome as a single Page which I don't really want to do. I didn't __init__ because I wasn't doing anything to the Page object that required it. Inheritance from PyPOM's constructor was good enough.